benchmark/streamly-benchmarks.cabal

```diff
@@ -130,7 +133,12 @@ library
     import: lib-options, bench-depends
     hs-source-dirs: lib
     exposed-modules: Streamly.Benchmark.Common
+                   , Streamly.Benchmark.Common.Handle
                    , Streamly.Benchmark.Prelude
+    build-depends:
+          typed-process     >= 0.2.3 && < 0.3
+        , directory         >= 1.2.2 && < 1.4
+        , ghc-prim          >= 0.4   && < 0.7 
```

Comment:
These are already in `bench-depends`, so we can remove these?

Reply:
I had thought library dependencies need to be specified separately from benchmark dependencies. Removed now.